### PR TITLE
[DOCs/operators]: Test WSS for exit-gateway, write a tutorial & update reversed proxy page

### PR DESCRIPTION
**Pull request by serinko**
Another tool for operators to secure their nodes and display their landing pages for public/ISP etc.

### CODE REVIEW

File: documentation/operators/src/nodes/proxy-configuration.md [hunk 1]
```diff
@@ -374,6 +378,261 @@ where you should read out
...
 or just point your browser to the URI which you set above, such as https://nym-exit.<YOUR_DOMAIN>
 
+
+
+
+```
+
+## Web Secure Socket Setup
+
+- INTRO WHY
```
Comment (mfahampshire) [inline documentation/operators/src/nodes/proxy-configuration.md]: Is this a comment left in or is the page WIP?

File: documentation/operators/src/nodes/proxy-configuration.md [hunk 2]
```diff
@@ -374,6 +378,261 @@ where you should read out
...
+We documented two options for node operators to setup WSS for `nym-node`:
+
+1. [Using a script](#using-a-script)
+2. [Step by step](#step-by-step)
+
+**Variables Explanation**
+
+This guide contains several variables. Substitute them with your own value, without `<>` brackets. Here is a list of variables we used below.
+
+| Variable              | Description                                                    | Syntax example                                            |
```
Comment (mfahampshire) [inline documentation/operators/src/nodes/proxy-configuration.md]: Nice this is a cool approach. Maybe we can do something like this as a standalone page for the operators docs as well, a 'variable glossary'

File: documentation/operators/src/nodes/proxy-configuration.md [hunk 3]
```diff
@@ -374,6 +378,261 @@ where you should read out
...
+```
+
+Your `nym-node` should be configured to run over WSS now. You can do a few quick checks to test that the setup worked out.
+
+- `wscat -c ws://<IP>:<WSS_PORT>`
+
+### Step by Step Guide
+
+
 <!--
```
Comment (mfahampshire) [inline documentation/operators/src/nodes/proxy-configuration.md]: Is this a comment that was left in?

**Review by mfahampshire — CHANGES_REQUESTED** [review]

File: documentation/operators/src/nodes/proxy-configuration.md (hunk 3 above)
Comment (serinko) [reply documentation/operators/src/nodes/proxy-configuration.md]: Oh I wasn't finished, I should probably label it as draft meanwhile, I am sorry, but I will address your comments meanwhile, thank you.

**Review by mfahampshire — APPROVED** [review]

**Pull request merged**